web/modules/custom/dpl_webform_configuration/dpl_webform_configuration.module

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for dpl_webform_configuration module.
+ */
+
+use Drupal\webform\Entity\Webform;
+
+/**
+ * Implements hook_ENTITY_TYPE_create() for webform.
+ */
+function dpl_webform_configuration_webform_create(Webform $webform): void {
+  // Disable saving of results for all new webforms.
```

Comment:
Great with comment. I think you should add why we are disabling the saving.
Its always good to write the WHY in comments and in git commit massges.
I keep forgetting it myself 😆

Reply:
I agree! I have added some more informative commenting now.

I am also trying to be better to do good commit messages, but practice takes time :D